### DISCUSSION TITLE
Define the minimum requirements for a media element

### DISCUSSION THREAD

**Discussion by dylanjha** [muxinc/media-chrome#428] [Proposals]
1. The entire Video element API not not required to for a media elements, let's define explicitly what subset is required (maybe a `media-element.md` file in docs/) ?
2. Let's also make sure we have some test coverage to make sure a minimal video element works on the normal happy paths

Discussion happened in this thread: https://github.com/muxinc/media-chrome/pull/93#discussion_r752524794

**Comment by cjpillsbury** [comment]
Should we start this in https://github.com/muxinc/media-chrome/discussions/categories/proposals ?
